deploy/backplane/10-cee-read.ClusterRole.yml

```
@@ -0,0 +1,18 @@
+aggregationRule:
+  clusterRoleSelectors:
+    # aggregate all "view" scope rbac
+    - matchExpressions:
+        - key: rbac.authorization.k8s.io/aggregate-to-view
+          operator: In
+          values:
+            - "true"
+    - matchExpressions:
+        - key: managed.openshift.io/aggregate-to-backplane
```

This label isn't very clear.  Rather we have a discrete label per role it's aggregated into.
```suggestion
        - key: managed.openshift.io/aggregate-to-backplane-cee
```

---

agreed, will make the change for both aggregated roles. I wasnt sure of the naming convention of the label